templates/minimal/README.md

```diff
@@ -47,19 +47,64 @@ cd minimal-template
 
 ### Build
 
-🔨 Use the following command to build the node without launching it:
+🔨 Use the following command to build the template, which by default
+compiles just the runtime. There is also a `parachain-template-node`
+crate that is able to run and load the runtime but the recommended
+way of starting the template is based on `Omni Node`.
 
 ```sh
 cargo build --release
 ```
 
-🐳 Alternatively, build the docker image:
+### Single-node Development Chain with Omni Node
+
+⬇️  Omni Node can be run by using the `polkadot-omni-node` binary, which can be
```

Comment:
Better, more wholesome suggestion: Add a `README.md` to `polkadot-omni-node` that has a good how to install section. And, you can refer to it there. 

Options: 

1. download + add to PATH
2. `cargo install polkadot-omni-node`

You can write this README as-if the next release is already out.

Reply:
Both here and within the new README, you can backlink to 

https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_docs/reference_docs/omni_node/index.html

This is a principle that we try to follow in our (rust)docs, and to try and minimize repetition (within reason, of course), and backlink where possible.

Reply:
Regarding download, we are waiting for https://github.com/paritytech/polkadot-sdk/issues/802

Reply:
All in all, you can set your gaze at making sure the following have good READMEs: 

* https://crates.io/crates/frame-omni-bencher
* https://crates.io/crates/staging-chain-spec-builder (in which please note that https://crates.io/crates/chain-spec-builder is NOT us)
* https://crates.io/crates/polkadot-omni-node

all have a baseline quality, and are consistent, because they will appear in `crates.io` which is important for binary crates. 

The best way to get there is to use the tools yourself, and see how you would write a README for someone else. You have already started doing this. 

It is up to you, but I suggest writing all these README's as `//!` in the `main.rs`, and then using `cargo-readme` to generate READMEs from them.

I think this can be a reasonable follow-up to this, or if you prefer to do it all in this PR, I would be happy to review in one place.

Reply:
>  Add a README.md to polkadot-omni-node that has a good how to install section

Great suggestion, wrote the README here with the installation options: [6b17432](https://github.com/paritytech/polkadot-sdk/pull/5954/commits/6b1743206e4c6d1f1a5185190e75fe427e39b7c9).

> Regarding download, we are waiting for https://github.com/paritytech/polkadot-sdk/issues/802

Waiting for #802 feels a bit like scope creep. My understanding is that the release is pending only on #6057. The commit linked above with omni-node README references a download section which is speculative (and was crafted based on how `polkadot-parachain` binary download looks like). I mean, #802 is still important, but it might need to address not just `polkadot-omni-node` binary, and it might take more than wanted (I might be wrong though and things are simpler than they seem). I can update the README to reflect ARM binaries download too once I'll see how these artifacts can look like on the releases page.

> All in all, you can set your gaze at making sure the following have good READMEs:
> I think this can be a reasonable follow-up to this, or if you prefer to do it all in this PR, I would be happy to review in one place.

I updated all of them in this PR. I added a README for chain-spec-builder (starting from a `cargo readme` run in the crate root):  [c2cc3f6](https://github.com/paritytech/polkadot-sdk/pull/5954/commits/c2cc3f6ae71097d8902e65be4ae8b8f66ad5296a). I updated `polkadot-omni-node` README a bit because I felt the wording can be better. Overall all crates' READMEs follow mostly the same structure ( title, installation, usage, and some misc section at the end).

## Final thoughts

All these docs will be referenced to some extent in the templates docs. Will follow up soon with the templates updates.